lbl-srg/ctrl-flow-dev
# vender specific annotation

### Description
This adds a deadEnd attribute to the InputGroup when an InputGroupExtend class element has the vender specific annotation.

### Related Issue(s)
#96 

### Testing
Use the debug tree script in order to see if children on the extend class are not set to visible.
Steps:

1. Start up the server in a container. 
2. Update Directory location for the templates.json creation. `server/scripts/parse-template-package.ts` `${__dirname}/templates.json`
3. Generate the tempates.json file with the following script: `npm run parseTemplateJSON`
4. Run the following script from the script folder: `node visualize-option-tree.js`
5. View the output.html in a browser.
